index.js

```diff
@@ -120,9 +120,13 @@ class NodeTestHelper extends EventEmitter {
         logSpy.METRIC = log.METRIC;
         this.log = () => logSpy;
```

Comment:
returns a function that returns a spy, when it should return the log data... I think.

Reply:
Nope, that's not it.  We're just not getting the log data expected.

Reply:
It's possible there's a sinon/should version disagreement

Reply:
tomorrow, I'll try to confirm.  I've been doing my testing against `node-red-nodes`, which *does* use this module.

Reply:
Sounds good - thanks.  I'd like to make sure it works against the core, or at least know how to change the core tests if necessary so we can use it when ready.